cli.js

```diff
@@ -274,30 +307,23 @@ async function withdraw({ deposit, currency, amount, recipient, relayerURL, refu
     }
   } else {
     // using private key
-    const { proof, args } = await generateProof({ deposit, recipient, refund })
+
+    // check if the address of recepient matches with the account of provided private key from environment to prevent accidental use of deposit address for withdrawal transaction.
+    const { address } = await web3.eth.accounts.privateKeyToAccount('0x' + PRIVATE_KEY)
+    assert(recipient.toLowerCase() == address.toLowerCase(), 'Withdrawal amount recepient',recipient,'mismatches with the account of provided private key from environment file',address)
```

Comment:
This will prevent users using same account for deposit & withdrawal transactions.